docs/content/querying/lookups.md

```diff
@@ -0,0 +1,191 @@
+# Lookups
+
+Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value.
```

Comment:
technically dim extractions do this as well and this isn't just unique to lookups